src/RepoFrontend.ts

```diff
@@ -165,7 +168,8 @@ export class RepoFrontend {
     this.toBackend.push({ type: 'Query', id, query })
   }
 
-  open = <T>(url: DocUrl): Handle<T> => {
+  open = <T>(url: DocUrl, crawl: boolean = true): Handle<T> => {
```

Comment:
I don't love this usage of `crawl` as an arg. Any alternative ideas welcome.

Reply:
I'd just cut it for now btw. YAGNI. When we find out what we need, we can add it.

Reply:
we need it to prevent infinite loops while crawling: the crawler calls open, which would call crawl, which would call open, ad infinitum. We need some way of managing this, and this boolean arg gets the job done - I just don't love it.